docker/iceberg-rest-adapter-image/Dockerfile

```diff
@@ -0,0 +1,44 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+FROM azul/zulu-openjdk:17 as builder
+
+WORKDIR /app/
+COPY open-api/build/libs/iceberg-open-api-test-fixtures-runtime-*.jar /app/
```

Comment:
> Do the actual build inside this container

I tried to add following before.

```
FROM azul/zulu-openjdk:17 as builder

WORKDIR /app
COPY . .
RUN ./gradlew :iceberg-open-api:shadowJar

```

a) Copying whole Iceberg diretory takes time, showed 1.61 GB even after gradle clean
b) Has to download gradle (that took lot of time)
c) Compile failed as
   " Neither version.txt nor git version exists: Git not found in project"

Reply:
> Remove this stage, since it doesn't add any value. In line 23 we copy the jar into the container, and at line 33 we copy it from the builder container into the runner container.

updated.